packages/bpk-component-calendar/src/BpkCalendarDate.scss

```diff
@@ -60,14 +60,14 @@
     cursor: default;
 
     @include bpk-themeable-property(background-color, --bpk-calendar-date-selected-background-color, $bpk-state-selected-background-color);
-    @include bpk-themeable-property(color, --bpk-calendar-date-text-selected-color, $bpk-calendar-day-selected-color);
+    @include bpk-themeable-property(color, --bpk-calendar-date-text-selected-color, $bpk-color-white);
   }
 
   &--focused:not(:disabled):not(.bpk-calendar-date--selected) {
-    box-shadow: 0 0 0 2px $bpk-state-selected-background-color inset;
-    box-shadow: 0 0 0 2px var(--bpk-calendar-date-focused-border-color, $bpk-state-selected-background-color) inset;
```

Comment:
can we keep this mixin and update the value in `packages/bpk-tokens/src/web/base/states.json`